examples/nodejs-server/.app-config.schema.yml

```diff
@@ -1,11 +1,34 @@
+$schema: http://json-schema.org/draft-07/schema#
```

Comment:
this is inferred by default, though is this for vs code?

Reply:
It worked fine without this, I just wanted to be verbose and include it for clarity. 
Removed as it is unnecessary.